planner/core/testdata/integration_serial_suite_out.json

```diff
@@ -51,7 +540,7 @@
           "HashAgg_20 1.00 root  funcs:count(Column#12)->Column#11",
           "└─TableReader_21 1.00 root  data:HashAgg_8",
           "  └─HashAgg_8 1.00 cop[tiflash]  funcs:count(1)->Column#12",
-          "    └─BroadcastJoin_11 8.00 cop[tiflash]  inner join, left key:test.fact_t.d1_k, right key:test.d1_t.d1_k",
```

Comment:
many of broadcastjoins are changed to be hash join,  is it ok?

Reply:
There name has been changed, but the operators keep same.